test/deployframework/context.go

```diff
@@ -111,14 +160,22 @@ func (ctx *DeployerCtx) Setup() (*reportingframework.ReportingFramework, error)
 
 	ctx.Logger.Infof("Report results directory: %s", reportResultsPath)
 
-	// TODO create functions that determine the reportingframework fields
-	// we can hardcode these values for now as we only have one
-	// meteringconfig configuration that gets installed and we dont
-	// support passing a METERING_CR_FILE yaml file for local testing
-	useHTTPSAPI := true
-	useRouteForReportingAPI := true
-	useKubeProxyForReportingAPI := false
-	reportingAPIURL := ""
+	if ctx.RunTestLocal {
+		useHTTPSAPI = false
+		useRouteForReportingAPI = false
+		useKubeProxyForReportingAPI = false
+		reportingAPIURL = fmt.Sprintf("%s://%s:%d/healthy", httpScheme, localAddr, apiPort)
```

Comment:
This still didn't get renamed to reflect that it is not a reporting API URL, just a health endpoint.